src/setup/destroy.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  DeleteBucketCommand,
+  DeleteObjectCommand,
+  ListObjectsCommand,
+} from "@stedi/sdk-client-buckets";
+import { bucketClient } from "../lib/buckets.js";
+import { guidesClient } from "../support/guide.js";
+import { stashClient } from "../lib/stash.js";
+import { DeleteGuideCommand } from "@stedi/sdk-client-guides";
+import {
+  PARTNERS_KEYSPACE_NAME,
+  CONTROL_NUMBER_KEYSPACE_NAME,
+} from "../lib/constants.js";
+import { DeleteKeyspaceCommand } from "@stedi/sdk-client-stash";
+import * as dotenv from "dotenv";
+import { DeleteUserCommand } from "@stedi/sdk-client-sftp";
+import { sftpClient } from "../lib/sftp.js";
+import { functionClient } from "../support/functions.js";
+import { DeleteFunctionCommand } from "@stedi/sdk-client-functions";
+import { updateDotEnvFile } from "../support/utils.js";
+dotenv.config();
+
+(async () => {
+  console.log("Deleting all resources provisioned by bootstrap");
+
+  // Delete Buckets
+  console.log("Deleting Buckets");
+  await emptyAndDeleteBucket(process.env.SFTP_BUCKET_NAME ?? "");
```

Comment:
nit: could use `requiredEnvVar()` here to avoid the fallback empty strings if you want

Reply:
didn't catch this while pairing, but yeah, `requiredEnvVar` sounds good here 😂